apps/server/lib/lexical/server/code_intelligence/completion/translations/macro.ex

```diff
@@ -438,6 +438,62 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Macro do
     |> builder.boost()
   end
 
+  @stub_label ~S(test "message"           )
+  @plain_label ~S(test "message" do...     )
+  @context_label ~S(test "message", %{} do...)
+
+  def translate(%Candidate.Macro{name: "test", arity: 1}, builder, env) do
+    stub_label = @stub_label
+
+    stub_snippet = """
+    test "${0:message}"
+    """
+
+    env
+    |> builder.snippet(stub_snippet,
+      detail: "A stub test",
+      kind: :class,
+      label: stub_label
+    )
+    |> builder.boost(1)
+  end
+
+  def translate(%Candidate.Macro{name: "test", arity: 2}, builder, env) do
+    plain_label = @plain_label
+
+    plain_snippet = """
+    test "${1:message}" do
+      ${0:body}
```

Comment:
```suggestion
      $0
```
I think adding body is a little annoying, it's not syntactically valid, and if you miss a tab stop, it causes an error.